src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -420,6 +420,16 @@ public RemoteBuildEventUploadModeConverter() {
               + " --experimental_remote_cache_compression_threshold.")
   public boolean cacheCompression;
 
+  @Option(
+      name = "remote_use_output_paths",
```

Comment:
Since this is more like a guard for the incompatible change than a feature, can we rename it to `--incompatible_remote_use_output_paths`? It's likely we will remove this flag in near future.

Reply:
done